engine-comparison/dispatcher.sh

```diff
@@ -66,6 +66,17 @@ build_benchmark_using() {
   mkdir $SEND_DIR
 
   cp ${BUILDING_DIR}/${BENCHMARK}-${FUZZING_ENGINE} $SEND_DIR
+
+  #TODO: make these gsutil cp?
```

Comment:
I was thinking both locations could be on `gs://`, because things like `runner.sh` were placed there by `begin-experiment` long ago